python/hail/dataset.py

```diff
@@ -2152,7 +2130,7 @@ def filter_samples_expr(self, expr, keep=True):
         - ``s`` (*Sample*): sample
         - ``sa``: sample annotations
         - ``global``: global annotations
-        - ``gs`` (*Aggregable[Genotype]*): aggregable of :ref:`genotype` for sample ``s``
+        - ``gs`` (*Aggregable[Genotype]*): aggregable of genotype for sample ``s``
```

Comment:
aggregable[genotype] is wrong

Reply:
this is found a bunch of places, actually.

Reply:
Maybe 
```
- ``gs`` (*Aggregable[Struct]*): aggregable of genotype structs for sample ``s``
```

Reply:
It doesn't have to be a struct.  I'm trying to sloppily use "genotype" to mean whatever the cell in the matrix is so I don't have to revise all the documentation when it is going to get revised again for the new python API.  Another idea: I could still write `Aggregable[genotype]`, but have the genotype link out to `genotype_schema`.

Reply:
Alright, yeah. Don't worry about this.